paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentConfirmationOption.kt

```diff
@@ -13,6 +13,22 @@ internal sealed interface PaymentConfirmationOption : Parcelable {
         val optionsParams: PaymentMethodOptionsParams?,
     ) : PaymentConfirmationOption
 
+    @Parcelize
+    data class GooglePay(
+        val config: Config,
+    ) : PaymentConfirmationOption {
+        @Parcelize
+        data class Config(
+            val environment: PaymentSheet.GooglePayConfiguration.Environment?,
+            val merchantName: String,
+            val merchantCountryCode: String,
+            val merchantCurrencyCode: String?,
+            val customAmount: Long?,
+            val customLabel: String?,
+            val billingDetailsCollectionConfiguration: PaymentSheet.BillingDetailsCollectionConfiguration,
+        ) : Parcelable
```

Comment:
I would love to inject these values but `FlowController` can't inject these values at the moment because it requires the user to manually configure first.